### ISSUE TITLE
Read the Ripples (RtR) Quest Script Wanted!

### ISSUE THREAD

**Issue by jandersson** [rpherbig/dr-scripts#4289]
Would be nice to have a script that completes the RtR quest or at least part of it.

This quest isn't very difficult, it is just time consuming and a death-prone.

Requirements:
* The script should fill all your prediction pools too 100% in one session. `astrology.lic` could also contain this logic though. I think this is a specific use case warranting a separate script. Do not fill the event pool, since it is irrelevant for this quest!
* The script should ensure `afk.lic` is running and start it if not. Since you have a good chance of dying, this will help prevent item loss.
* The script should wear your standard armor gear via EquipmentManager since there are mobs floating around in the quest area
* The script should disable the sanowret crystal script and wait for 100% concentration before starting the ritual. Wait for 100% mana and health while we're at it. Shit, go get healed too.
* The script should perform the ritual to get the spell

Since you can only complete this once, its gonna be hard to test. I think a good start will be to implement all requirements except for the ritual step and fill that in later.

**Comment by cfrisby77** [comment]
has anyone done in work on this, i have a moonie at 40 and should be able to test soon.
